cli/src/main/java/org/opentosca/toscana/cli/commands/Constants.java

```diff
@@ -0,0 +1,87 @@
+package org.opentosca.toscana.cli.commands;
+
+/**
+ Strings that are used in the CLI, for easier test usage declared here
+ */
+public final class Constants {
+
+    public final String CSAR_UPLOAD_SUCCESS = "Upload of CSAR Archive was successful!";
+    public final String CSAR_UPLOAD_ERROR = "Something went wrong while uploading the CSAR.";
+    public final String CSAR_UPLOAD_ERROR400 = "Parsing of the CSAR failed.";
+    public final String CSAR_UPLOAD_ERROR400M = "A CSAR with given name already exists.\n";
+    public final String CSAR_UPLOAD_ERROR500 = "Processing failed.\n";
+    public final String CSAR_DELETE_SUCCESS = "Deletion of CSAR was successfull!";
+    public final String CSAR_DELETE_ERROR = "Something went wrong while deleting the CSAR.";
+    public final String CSAR_DELETE_ERROR404 = "CSAR not found!";
+    public final String CSAR_DELETE_ERROR404M = "CSAR not found!\n";
+    public final String CSAR_DELETE_ERROR500 = "Deletion of CSAR has failed!";
+    public final String CSAR_DELETE_ERROR500M = "Deletion of CSAR has failed!\n";
+    public final String CSAR_LIST_SUCCESS = "List of CSARs:";
+    public final String CSAR_LIST_EMPTY = "List of CSARs is empty.";
+    public final String CSAR_LIST_ERROR = "Something went wrong while getting the CSAR List.";
+    public final String CSAR_INFO_SUCCESS = "Informations about the CSAR:\nName: ";
```

Comment:
Information is uncountable and therefore, you would always say: "Information about the csar: ..."